packages/metascraper-iframe/src/from-twitter.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const { normalizeUrl, memoizeOne } = require('@metascraper/helpers')
+const { map } = require('lodash')
+
+const getPlayerUrl = memoizeOne(
+  (url, $) =>
+    normalizeUrl(
+      url,
+      $('meta[name="twitter:player"]').attr('content') ||
+        $('meta[property="twitter:player"]').attr('content') ||
+        $('meta[name="twitter:player:stream"]').attr('content') ||
+        $('meta[property="twitter:player:stream"]').attr('content')
```

Comment:
@nathangathright I added `twitter:player:stream` as well 👍

Reply:
`twitter:player:stream` _should_ point to the MP3/MP3 file URL if available, so I don't think that's what you want to be inserted as the source for an iframe unless you add a contentType check. It's already being pulled into metascraper-audio and metascraper-video correctly.

In fact, the references to `twitter:player` in the [audio](https://github.com/microlinkhq/metascraper/blob/52d8598415802e92e10fcdffc4bc21aa28a640ea/packages/metascraper-audio/index.js#L34-L35) and [video](https://github.com/microlinkhq/metascraper/blob/b1e27626ad38d6efbc24f5104df49b2d9be3631e/packages/metascraper-video/index.js#L37-L38) packages _should_ never pass the `isMime` check if everyone follows the spec, but it doesn't hurt to leave them there.

Reply:
Thanks for that comment! I the values there. Can you double-check my last two commits?

I prefer to keep the core as restrict as possible, that will be benefit strict markup 🙂

Reply:
@nathangathright you are definitively right since I'm checking for audio/video source inside the player:

https://github.com/microlinkhq/metascraper/blob/2ff8d72be47ab4204e7753123d15f3d2805e2fee/packages/metascraper-audio/index.js#L63

Reply:
Looks good to me!